### PR TITLE
PPU LLVM: improve accuracy of VSL/VSR

### PR DESCRIPTION
**Pull request by Nekotekina**
Got annoyed by false negatives in ppu_vpu test.

### CODE REVIEW

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -330,6 +330,28 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 
 	m_emu_settings->EnhanceRadioButton(ppu_bg, emu_settings_type::PPUDecoder);
 
+	connect(ppu_bg, &QButtonGroup::idToggled, [this](int id, bool checked)
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp:333]: Ugh... Need to put this before EnhanceRadioButton

Comment (Nekotekina) [reply rpcs3/rpcs3qt/settings_dialog.cpp:333]: Damn. Why though? It seems working.

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp:333]: initial state

**Comment by coolllman** [comment]
Call of duty not work with this pr, when start stage result black screen, on master 13185 all good

**Pull request merged**